src/lib/storage/fixed_width_integer_vector.cpp

```diff
@@ -0,0 +1,40 @@
+#include "fixed_width_integer_vector.hpp"
+#include "all_type_variant.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+template <typename T>
+FixedWidthIntegerVector<T>::FixedWidthIntegerVector(size_t size) {
+  _attribute_vector = std::vector<T>(size);
+}
+
+template <typename T>
+ValueID FixedWidthIntegerVector<T>::get(const size_t index) const {
+  DebugAssert(index < _attribute_vector.size(), "Index out of bounds");
+  return ValueID{_attribute_vector[index]};
+}
+
+// Sets the value id at a given position.
+template <typename T>
+void FixedWidthIntegerVector<T>::set(const size_t index, const ValueID value_id) {
```

Comment:
I think it is more common to put documentation comments at the declaration (that is, in the header file)